test/__fixtures__/override-method/output.js

```diff
@@ -0,0 +1,5 @@
+Ext.override('Ext.panel.Panel', {
+    myMethod: function () {
+        (Ext.panel.Panel.prototype || Ext.panel.Panel).myMethod.call(this);
```

Comment:
This will be a problem by the looks of things, as this resultant code will cause an infinite loop by calling itself recursively (because the override is actually overwriting `Ext.panel.Panel.myMethod`, calling `Ext.panel.Panel.myMethod` just invokes this new method rather than the previous one).

This is why for the Ext.define/override behaviour, I create a variable which stores a reference to the previous method _before_ overriding, and then invoke that in the replacement call.

The behaviour for this is handled in `getOverrideMethodRef`, which should be okay to reuse for this purpose.

Reply:
True, I've not thought of that, makes sense now! The problem is addressed in the next commit. I also updated the Readme.md to reflect those changes.